frontend/javascripts/admin/api/jobs.ts

```diff
@@ -208,6 +208,8 @@ function startSegmentationAnnotationDependentJob(
   annotationId: string,
   annotationType: APIAnnotationType,
   mergeSegments?: boolean,
+  includesProofreading?: boolean,
+  selectedBoundingBox?: Vector6,
```

Comment:
"selected" sounds as if this variable would refer to a bounding box (ID) selected in the UI. I'd rename it to simply `boundingBox`.